src/common/utils/validateDuplicateFieldValue.js

```diff
@@ -18,6 +18,8 @@ export const validateDuplicateFieldValue = async ({
     return errorRequired;
   }
 
+  if (fieldValue?.includes(':')) return <FormattedMessage id="ui-finance.validation.mustNotIncludeColon" />;
```

Comment:
is it expected that ledger and FY will have the same colon restriction?

Reply:
Oops 🙂